src/current/_includes/releases/v23.1/v23.1.17.md

```diff
@@ -0,0 +1,106 @@
+## v23.1.17
+
+Release Date: March 19, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-17-security-updates">Security updates</h3>
+
+- A user with the `VIEWACTIVITY` privilege can now request a statement bundle from the [**Statements** page](https://www.cockroachlabs.com/docs/v23.1/ui-statements-page) in the DB Console or with the internal builtin function `crdb_internal.requets_statement_bundle`. [#119638][#119638]
+- Clusters using [Cluster Single Sign-on (SSO) using JSON web tokens (JWTs)](https://www.cockroachlabs.com/docs/v23.1/sso-sql) can now optionally fetch signing keys from configured issuers instead of configuring static signing keys for each issuer. When new cluster setting `server.jwt_authentication.jwks_auto_fetch.enabled` is set to `true`, signing keys are automatically fetched from the issuer using metadata published in its OpenID configuration. In this case, static signing keys in `server.jwt_authentication.jwks` are ignored. When automatch fetching is enabled, there may be a slight increase network latency in each JWT authentication request, proportional to the latency between the cluster and the issuer's endpoint. [#120063][#120063]
```

Comment:
```suggestion
- Clusters using [Cluster Single Sign-on (SSO) with JSON web tokens (JWTs)](https://www.cockroachlabs.com/docs/v23.1/sso-sql) can now optionally fetch signing keys from configured issuers instead of configuring static signing keys for each issuer. When the new cluster setting `server.jwt_authentication.jwks_auto_fetch.enabled` is set to `true`, signing keys are automatically fetched from the issuer using metadata published in its OpenID configuration. In this case, static signing keys in `server.jwt_authentication.jwks` are ignored. When automatic fetching is enabled, there may be a slight increase in network latency for each JWT authentication request, proportional to the latency between the cluster and the issuer's endpoint. [#120063][#120063]
```